packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2305,6 +2307,20 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin, Resto
     }
   }
 
+  /// Close Drawer if opened if it is currently opened.
```

Comment:
```suggestion
  /// Closes [Drawer] if it is currently opened.
```

Reply:
can you also add more doc around how to get the ScaffoldState like `openDrawer` and `openEndDrawer`?